docs/src/modules/components/AppDrawer.js

```diff
@@ -163,7 +163,7 @@ function AppDrawer(props) {
       </div>
       <Divider />
       <Box mx={3} my={2}>
-        <DiamondSponsors />
+        <DiamondSponsors spot="drawer" />
```

Comment:
Thanks, was planned in the next batch :) https://trello.com/c/Eifr9U6q/2580-diamondsponsors-add-spotsidenav

Reply:
No worries. Was just a bit irritating when investigating StrictMode warnings  :smile:

Reply:
So do you prefer "sidenav"? I don't think we used this name before.

Reply:
I have used `sidenav` in the first report shared to Sencha. No specific preference.